incubator/nfs/templates/nfs-server-pv.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: nfs-pv-provisioning-demo
+  labels:
+    demo: nfs-pv-provisioning
+  annotations:
+    volume.alpha.kubernetes.io/storage-class: any
```

Comment:
Use `storageClassName` instead.

Reply:
Noted.  I still owe some other fixes to make this chart more standard and best-practices.  Thanks again for your feedback, I'm sorry holidays got me busy!